cmd/scaffold.go

```diff
@@ -1083,11 +1091,20 @@ func (fnb *FlowNodeBuilder) onStart() error {
 // postShutdown is called by the node before exiting
 // put any cleanup code here that should be run after all components have stopped
 func (fnb *FlowNodeBuilder) postShutdown() error {
+	var errs *multierror.Error
+	for _, fn := range fnb.postShutdownFns {
+		err := fn()
+		if err != nil {
+			errs = multierror.Append(errs, err)
+		}
+	}
+
 	err := fnb.closeDatabase()
 	if err != nil {
```

Comment:
nit: We can probably implement this closeDatabase() as a special ShutdownFunc inside scaffold.

Reply:
updated fde01f5